homeassistant/components/squeezebox/media_player.py

```diff
@@ -262,6 +266,7 @@ def __init__(self, coordinator: SqueezeBoxPlayerUpdateCoordinator) -> None:
         self._previous_media_position = 0
         self._attr_unique_id = format_mac(self._player.player_id)
         self._browse_data = BrowseData()
+        self._synthetic_media_browser_thumbnail_items: LRU[str, str] = LRU(5000)
```

Comment:
5000 seems excessive, and won't ever be used that way. It's only for apps and radio, so we need like 100 ?

Reply:
Actually, this is a bit misleading, because it's not just for the app overview, but also for every single clickable element within an app.
Spotify for example is an app and each menu option, album and track in there needs such a synthetic ID to have a thumbnail image in the HA media browser.

In a quick test, by opening my favorites + a bunch of playlists, I now have over a thousand entries in that LRU.
So 5000, while still large, isn't actually as oversized as it might sound.